docs/shared-content-source/docs/modules/develop/pages/use-flink-streamlets.adoc

```diff
@@ -200,14 +200,14 @@ lazy val yourStreamlet = appModule("yourStreamlet")
       sbtdocker.Instructions.User("root"),
       sbtdocker.Instructions.Run("mkdir -p ./plugins/azure-fs-hadoop"),
       sbtdocker.Instructions.Run("cp /opt/flink/opt/flink-azure-fs-hadoop-1.10.0.jar ./plugins/azure-fs-hadoop/"),
-      sbtdocker.Instructions.Run("chown -R flink:flink ./plugins/azure-fs-hadoop"))
-  )
+      sbtdocker.Instructions.Run("chown -R flink:flink ./plugins/azure-fs-hadoop"),
+      sbtdocker.Instructions.User("cloudflow")))
   ...
 ----
 
```

Comment:
You do not need to muck around with docker. just add libraries, as in local case and they will make it to the docker

Reply:
I think so too, please test this.

Reply:
Justin, can you, please, test this

Reply:
Please also refer to https://ci.apache.org/projects/flink/flink-docs-stable/ops/filesystems/ since there are some comments there about plugins and lib directory.

Reply:
Thanks @blublinsky for testing this works without the additional Docker commands. Will update shortly